src/components/Articles/ArticleList.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react"
+import axios from "axios"
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([])
+  const apiUrl = "https://backend-project-d9id.onrender.com/api/articles"
+
+  useEffect(() => {
+    const fetchArticles = async () => {
+      try {
+        const response = await axios.get(apiUrl)
+        setArticles(response.data.articles)
+      } catch (error) {
+        console.error("Error fetching articles:", error)
+      }
+    }
+
+    fetchArticles()
+  }, [apiUrl])
+
+  return (
+    <div className="article-list">
+      <h2>All Articles</h2>
+      {articles.map((article) => (
+        <div key={article.title} className="article-card">
```

Comment:
You could extract this code to the ArticleCard component which would then be rendered here for each article.